app/components/footer/FooterLink.vue

```diff
@@ -0,0 +1,22 @@
+<script setup lang="ts">
+import { NavigationMenuLink } from 'radix-vue'
+
+import type { NavigationLink } from '~/constants/header/navigationLinks.constant'
```

Comment:
Ik vind het vreemd dat de footer link een type gebruikt die uit `/constants/header` folder komt. Eventueel een folder `/navigation` folder maken?

Reply:
Hmm ik ga misschien een constants/footer maken? die links kunnen in principe anders zijn dan de header